src/shared_modules/utils/tests/socketDBWrapper_test.cpp

```diff
@@ -14,19 +14,17 @@
 #include "socketDBWrapper.hpp"
 
 // temp header
-#include "wazuhDBQueryBuilder.hpp"
 #include <chrono>
 #include <thread>
 
 TEST_F(SocketDBWrapperTest, EmptyTest)
 {
     m_query = "SELECT * FROM test_table;";
-    m_responses = std::vector<std::string> {""};
+    m_responses = std::vector<std::string> {" "};
```

Comment:
Why is there a space here? Aren't we testing an empty response?

Reply:
For the size of bytes to send :)